### DISCUSSION TITLE
Switch foot's dpi-aware to yes?

### DISCUSSION THREAD

**Discussion by heapifyman** [Manjaro-Sway/manjaro-sway#559] [Q&A] [answered]
foot version `1.15.0` changed the default for `dpi-aware` to `no` - see https://codeberg.org/dnkl/foot/releases/tag/1.15.0

This seems to lead to very tiny text size in foot terminal with manjaro sway's default font size of `:size=8` (at least if you don't use scaling).

Setting `dpi-aware=yes` in `foot.ini` seems to restore font size in foot to a readable size.

Alternatively, one can of course increase the `:size` in `foot.ini`.

Maybe `/usr/share/sway/templates/foot.ini` should set `dpi-aware=yes` or a bigger default font size?

**Comment by boredland** [comment, marked as answer]
I gave this a try. If `dpi-aware=yes`, the font in foot scales independently from the swaywm scale. I think it is a much better option to make foot not dpi-aware and adapt the font-size to your liking via the [userspace](https://github.com/manjaro-sway/manjaro-sway/blob/main/SUPPORT.md#how-can-i-customize-the-foot-terminal-without-losing-my-customization-after-an-upgrade) `foot.ini`. Just pass it something like `font=JetBrainsMono Nerd Font:size=11`. That way when scaling (even on multiple screens) is set in sway, the relative font size still makes sense.
